lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -76,6 +76,9 @@ end
 ---@field ret string
 local Function = {}
 Function.__index = Function
+Function.__tostring = function(self)
+	return "function(" .. self.arg_sig .. ")" .. (self.ret and ": " .. self.ret or "")
```

Comment:
```suggestion
	return "function(" .. self.arg_sig .. ")" .. (self.ret and (": " .. self.ret)) or ""
```

Reply:
This makes the LuaLS upset

Reply:
Need another set of parens wrapping `or ""` I assume